qa/suites/upgrade/octopus-x/parallel/1-ceph-install/octopus.yaml

```diff
@@ -43,6 +43,7 @@ tasks:
       - ceph orch ls
       - ceph orch host ls
       - ceph orch device ls
+      - ceph orch apply mds --placement=host.a
```

Comment:
that's missing a name here. the name of the cephfs.

Reply:
https://docs.ceph.com/docs/master/api/mon_command_api/#orch-apply-mds

Reply:
trying `     - ceph orch apply mds cephfs --placement=host.a`

Reply:
http://pulpito.ceph.com/teuthology-2020-06-29_20:00:13-upgrade:octopus-x:parallel-master-distro-basic-smithi/